src/components/CodeEditor.svelte

```diff
@@ -0,0 +1,186 @@
+<script>
+  import Prism from "prismjs";
+  import "prismjs/components/prism-kotlin";
+  import "prismjs/components/prism-swift";
+  import { onMount } from "svelte"; // eslint-disable-line import/no-extraneous-dependencies
+
+  export let codeTabs = [];
+
+  let currTab = codeTabs[0];
+  let showTooltip = false;
+  let codeHtmlOutput = "";
+
+  $: currLanguage = currTab.language.toLowerCase();
+
+  function updateCode() {
```

Comment:
I suppose this can be refactored as following:
```js
$: codeHtmlOutput = renderCode(currLanguage);

function updateCode(language) {
    // ...
}
```

By making `updateCode` a pure function, and using reactive assignment of Svelte, `onMount` call should no longer be needed.

Reply:
good advice